2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,18 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  const missingFromThePantry = {};
+  missingFromThePantry.name = recipe.name;
+  missingFromThePantry.items = [];
+    missingFromThePantry.items = recipe.ingredients.filter ((Ingredient) => {
```

Comment:
instead of declaring an empty object then adding to it we could directly declare the object.